itemv2.js

```diff
@@ -100,7 +120,12 @@ const BaseItem = {
 
 const BillItem = {
```

Comment:
Can we move this and `SimpleOption` to the bill file, keeping it all together. We can just have the base Item here, if we even still need it, they've diverged enough now that maybe they don't need to inherit the same props. That being said, consistent props like `name` and `amount` are nice.